kgforge/specializations/models/demo_model.py

```diff
@@ -96,8 +96,8 @@ def _validate_one(self, resource: Resource, type_: str) -> None:
     # Utils.
 
     @staticmethod
-    def _service_from_directory(dirpath: Path, context_iri: str, **dir_config):
-        return ModelLibrary(dirpath)
+    def _service_from_directory(dir_path: Path, context_iri: Optional[str]):
```

Comment:
(dir_path) This is changing the Model interface API. Unless really necessary, I would not rename this. Also any change on this method should also be done on the Model interface.

Reply:
It's not really necessary it's just to have consistent naming. What do you mean by the model interface API?